SleepEngine/main.cpp

```diff
@@ -11,9 +11,15 @@ int main()
 {
     EASY_PROFILER_ENABLE;
 
-	// right now the heap allocated objects leak because Updator does not own them, fixed in Updator re-do by tihran
-    auto initScene = [](slp::Game& game)
+	slp::Game game(WindowWidth, WindowHeight);
+
+    game.getClock().setFPS(60.f);
+	game.getClock().setRestrictFPS(true);
```

Comment:
```suggestion
    game.getClock().setRestrictFPS(true);
```
Fking tabs. Have no idea where it came from